test/student_test.py

```diff
@@ -0,0 +1,24 @@
+
+def student_test(uid, upwd):
+    auth = (uid, upwd)
+    link = 'http://localhost:8000/ttrs/students/signup/'
+
+    import requests
+    for i in range(1,6):
+        data = {}
+        data['username'] = 'stu{}'.format(i)
+        data['password'] = 'qwertyasdf'
+        data['email'] = 'stu{}@snu.ac.kr'.format(i)
+        data['grade'] = 1
+
+        from ttrs.models import College
+        data['college'] = College.objects.all()[0].id
+
+        print('Creating student {}...'.format(i))
+        res = requests.post(link, auth=auth, data=data)
```

Comment:
I think we should test the case that invalid data is came to the server. So, I recommend to refer to _Request & Response Example_ at the WIKI.

Reply:
Did you see this comment?

Reply:
Yes. I'm working on it.

Reply:
Okay, I'll wait.